main.cpp

```diff
@@ -56,10 +62,44 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
-    
+    int counter = 0; 
+    int people = 6;
+    int tries = 10;
+    bool flag = 0;
+    bool check = false;
+    bool signBit = 1;
+    float grade = 99.5f;
+    float degrees = 23.5f;
+    float input = 0.001f;
+    char initial = 'a';
+    char alphabetSoup = 'm';
+    char question = 'Q';
+    unsigned int peak = 10;
+    unsigned int positiveOnly = 1;
+    unsigned int rectifier = 100;
+    double timeLine = -100.003;
+    double blackHole = -29687543.001;
+    double antiGravity = 9999999999.99;
+
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(counter);
+    ignoreUnused(people);
+    ignoreUnused(tries);
+    ignoreUnused(flag);
+    ignoreUnused(check);
+    ignoreUnused(signBit);
+    ignoreUnused(grade);
+    ignoreUnused(degrees);
+    ignoreUnused(input);
+    ignoreUnused(initial);
+    ignoreUnused(alphabetSoup);
+    ignoreUnused(question);
+    ignoreUnused(peak);
+    ignoreUnused(positiveOnly);
+    ignoreUnused(rectifier);
+    ignoreUnused(timeLine);
+    ignoreUnused(blackHole);
+    ignoreUnused(antiGravity);
```

Comment:
pass all of your variables to a single call to ignoreUnused.
- [ ] revise

Reply:
I wish I could have written a program to put them in that function.  Woulda saved so much time!